<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntersectionGUIClass</class>
 <widget class="QMainWindow" name="IntersectionGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IntersectionGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QScrollArea" name="scrollArea3">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1100</width>
         <height>1036</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="canvas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFile"/>
   <addaction name="separator"/>
   <addaction name="actAddShape"/>
   <addaction name="actDelShape"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actSolve"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="FunctionArea">
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>805</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="functionArea">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>从文件添加</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="fileName"/>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnAddFile">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>添加图形</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QSpinBox" name="data1">
            <property name="minimum">
             <number>-100000</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>y1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="shapeType">
            <item>
             <property name="text">
              <string>直线</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>线段</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>射线</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>圆</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>y2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QSpinBox" name="data4">
            <property name="minimum">
             <number>-100000</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="data3">
            <property name="minimum">
             <number>-100000</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="data2">
            <property name="minimum">
             <number>-100000</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="tBtnAddShape">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="tBtnDelShape">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnClear">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSolve">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddShape">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actDelShape">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actAddFile">
   <property name="text">
    <string>文件添加</string>
   </property>
   <property name="toolTip">
    <string>文件添加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actSolve">
   <property name="text">
    <string>求解交点</string>
   </property>
   <property name="toolTip">
    <string>求解交点</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IntersectionGUI.qrc"/>
 </resources>
 <connections/>
</ui>
